### ISSUE TITLE
Bloodsail Buccaneers Reputation / Exalted Impossible.

### ISSUE THREAD

**Issue by GrizB37** [slowtorta/turtlewow-bug-tracker#4326]
**Please, provide a clear description what the bug is:**
The Devs have added alot of new quests and some more reputation items for this rep, rewards for hitting Revered and Exalted. 

The problem is, the quests only get you so far into honored after doing them all, I farmed Bruisers in Booty Bay for a 5-6 hours to get to Revered after doing all the quests. 

This is when the crazy part happens, they stop giving you rep once you hit Revered, and the only way to get to Exalted is to farm a vendor in Badlands for 5 rep per 5 minutes. 

Now if you ran in blind as I did and did all the quests first which give around 15k Rep, then you have to grind 21000 rep on this Badlands NPC, at 5 Rep per kill you need to kill this NPC 4200 times, on a 5 minute respawn timer, realistically this is like 4 weeks farming the only vendor in the badlands for Alliance. Even if you saved the quests for Revered you still need to kill the vendor 1200 times.

This is Very bad game design and I got up to 1000 rep before I got reported to the GM's for griefing NPC's. Which is fine, I agree this is not good. 

So all I'm asking is please let us grind to exalted Bloodsail Buccaneers Reputation by killing the Bruisers in Booty Bay, by enabled their rep gain up to exalted. Would still be a good grind but it doesn't inconvenience anyone killing guards unlike a vendor 4200 times. 

You've even added rep rewards for exalted albeit rubbish ones, but this doesn't matter for completionists / exalted rep chasers, so you must intend people to get exalted, but the only way is through griefing a vendor for weeks. 

TL:DR
Enable Bloodsail Buccaneers rep gain up until exalted from Booty Bay Bruisers, the only alternative is killing a vendor in Badlands 4200 times that people use alot and being told by the GM's to stop, making this rep grind Impossible.

**Comment by GrizB37** [comment]
Devs: We added Exalted rewards to Bloodsail Buccaneers!
GM: Do not kill that NPC Vendor repeatedly. (only one that gives rep)
Conculsion: Impossible to get the rewards/exalted
